Makefile

```diff
@@ -18,19 +18,16 @@ ifdef WERCKER
     TENANT := "oracle"
 else
     NEW_NAMESPACE ?= e2e-${USER}
-    VERSION := ${USER}-$(shell date +%Y%m%d%H%M%S)
+    VERSION := ${USER}-$(shell git describe --always --dirty)
     TENANT := "spinnaker"
 endif
 
 ROOT_DIR        := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))
 PKG             := github.com/oracle/mysql-operator
-REGISTRY        := iad.ocir.io/$(TENANT)
+REGISTRY        := iad.ocir.io
 SRC_DIRS        := cmd pkg test/examples
-REGISTRY_STRING := $(subst /,_,$(REGISTRY))
 CMD_DIRECTORIES := $(sort $(dir $(wildcard ./cmd/*/)))
 COMMANDS        := $(CMD_DIRECTORIES:./cmd/%/=%)
```

Comment:
No longer used

Reply:
Still used / referenced. Will remove in a subsequent PR when fixing the build step to keep things clean.